python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -1553,3 +1555,32 @@ def test_multiIndex_duplicate_names():
     )
 
     assert_eq(gi, pi)
+
+
+@pytest.mark.parametrize(
+    "names",
+    [
+        ["a", "b", "c"],
+        [None, None, None],
+        ["aa", "aa", "aa"],
+        ["bb", "aa", "aa"],
+        None,
+    ],
+)
+def test_pickle_rountrip_multiIndex(names):
+    df = cudf.DataFrame(
+        {
+            "one": [1, 2, 3],
+            "two": [True, False, True],
+            "three": ["ab", "cd", "ef"],
+            "four": [0.2, 0.1, -10.2],
+        }
+    )
+    actual_df = df.set_index(["one", "two", "three"])
+    actual_df.index.names = names
+    local_file = BytesIO()
+
+    pickle.dump(actual_df, local_file)
+    local_file.seek(0)
+    expected_df = pickle.load(local_file)
```

Comment:
I think you've switched the expected and actual here?

Reply:
Fixed it 👍